src/components/Question.tsx

```diff
@@ -226,18 +226,14 @@ const StyledQuestion = styled.div`
 
 export interface QuestionProps {
   question: QuestionType,
-  answer_id?: string;
-  task?: {
-    is_deleted: boolean;
-    type?: 'homework'
-  } | null,
-  correct_answer_id: ID;
+  answer_id?: ID;
+  task?: Task | null,
+  correct_answer_id: ID | null;
   incorrectAnswerId: ID;
   hideAnswers: boolean;
-  hidePreambles: boolean,
+  hidePreambles?: boolean,
   exercise_uid?: string;
   displayFormats: boolean,
-  processHtmlAndMath: boolean;
```

Comment:
After reviewing this prop on the Tutor side, it's actually supposed to be a function, but is never passed, so I'm removing it.